src/graphics/mod.rs

```diff
@@ -5,4 +5,6 @@ pub use style::*;
 mod graphic;
 pub use graphic::*;
 mod common;
+mod colour;
+
```

Comment:
```suggestion
mod colour;
pub use colour::*;
```
To be consistent with everything else in this file, see also https://github.com/25cst/libxdsim/pull/22#discussion_r2593191611